bgpd/bgp_vty.c

```diff
@@ -9359,7 +9359,9 @@ DEFUN (no_neighbor_addpath_tx_all_paths,
 	if (!peer)
 		return CMD_WARNING_CONFIG_FAILED;
 
-	if (peer->addpath_type[bgp_node_afi(vty)][bgp_node_safi(vty)] != BGP_ADDPATH_ALL) {
+	if (peer->addpath_type[bgp_node_afi(vty)][bgp_node_safi(vty) == SAFI_LABELED_UNICAST
```

Comment:
While this is okay functionally, can you please assign a separate variable to avoid these curls?

Reply:
Done. thanks for your help.